### ISSUE TITLE
Scroll in select, with load data on scroll

### ISSUE THREAD

**Issue by AymericSauty** [NG-ZORRO/ng-zorro-antd#7031]
### Reproduction link
[https://stackblitz.com/edit/ng-zorro-antd-ivy-bgz43x](https://stackblitz.com/edit/ng-zorro-antd-ivy-bgz43x)

### Steps to reproduce
Use the scroll bar of the nzSelect to scroll to the bottom of the select.
Then release click.

### What is expected?
I expect the scroll handler to be at the bottom of the select, where I release it.

### What is actually happening?
The scroll handler back to the top of the select, on the latest item hovered.

| Environment | Info |
|---|---|
| ng-zorro-antd | 12.0.1 |
| Browser | Chrome  |

**Comment by BBboy01** [comment]
> This is a bug, the behavior that you describe is ok. The problem is when you use the scroll bar of the nzSelect to scroll to the bottom of the select. If u make this clickly, the select came back to the last hovered item. so to the high top of the list.

You're right, it's just hard to believe this bug existed for so many years...

**Comment by skipah7** [comment]
@Laffery hello! Could you please look at this issue and PR? Seems like a simple fix for an old bug. Thanks!
